lib/auth/webauthncli/u2f.go

```diff
@@ -77,7 +78,10 @@ func RunOnU2FDevices(ctx context.Context, runCredentials ...func(Token) error) e
 		case errors.Is(err, errKeyMissingOrNotVerified):
 			// This is expected to happen a few times.
 		case err != nil:
-			log.WithError(err).Debug("Error interacting with U2F devices")
+			// suppress error spam, this error doesnt prevent u2f from working
+			if !strings.Contains(err.Error(), "hid: privilege violation") {
```

Comment:
+1 for also ignoring `hid: not permitted` unless that's actually a separate error that shouldn't be ignored.

Reply:
+1 for ignoring both errors too

Reply:
Right, forgot about `hid: not permitted` because I had my security settings configured to allow input monitoring. I'm actually surprised that one doesnt prevent u2f from working. Will ignore both then.